### ISSUE TITLE
Deep integration with react-native-skia (e.g. not via `ImageSVG` tag)

### ISSUE THREAD

**Issue by liamjones** [wuba/react-native-echarts#190]
**Is your feature request related to a problem? Please describe.**
This is related to the fact that labels, etc no longer render when the react-native-skia renderer is used as reported in issue https://github.com/wuba/react-native-echarts/issues/174.

We'd still like the option of rendering via the rn-skia library as it is much more performant than react-native-svg rendering.

**Describe the solution you'd like**
rn-echarts uses the rn-skia `Path`, `Polygon`, etc APIs to render the chart and isn't reliant on the `ImageSVG` tag.

**Describe alternatives you've considered**
For now, as a workaround, we've taken to moving to an old version of rn-skia that still renders text in Android but crashes on iOS and then rendering the chart with different renderers based on platform, see https://github.com/wuba/react-native-echarts/issues/174#issuecomment-2129350461 for details. I fully expect this to break entirely at some point in the future as RN moves forward. This also isn't a solution for anyone who still needs to use rn-skia for non echart related stuff on iOS.

Another alternative - fix the text rendering in rn-skia via the ImageSVG tag. The rn-skia maintainers have said they're not going to do it in https://github.com/Shopify/react-native-skia/issues/2404 / https://github.com/Shopify/react-native-skia/pull/2450. Doing this would require C++ knowledge.

**Additional context**

**Comment by zhiqingchen** [comment]
Yes, we have also considered such a solution, which is more in line with the principles of react-native-skia. It's just that there is not enough time to do it at the moment, but in the long run, we will consider this approach.

**Comment by troberts-28** [comment]
Massive +1 for this.

Great to hear that you're considering it @zhiqingchen. In the meantime, this needs to be more clearly acknowledged as a major limitation of the skia element of this library. The fact that you can use Skia is a huge performance benefit, but it's basically unusable without text rendering.

**Comment by zhiqingchen** [comment]
I completely agree. The performance benefits of using Skia are indeed significant, but the lack of text rendering capability is a major limitation

**Comment by zhiqingchen** [comment]
try 2.0.0-canary.3

**Comment by zhiqingchen** [comment]
https://wuba.github.io/react-native-echarts/docs/migration/v2

**Comment by liamjones** [comment]
Thank you zhiqingchen and 李志豪 for your work on this! I'm hoping to be able to try it out on our app soon.

**Comment by troberts-28** [comment]
Same here! Thank you both 🙏
